lib/portlayer/storage/nfs/disk.go

```diff
@@ -16,21 +16,28 @@ package nfs
 
 import (
 	"net/url"
+	"path"
 )
 
 //  Volume identifies an NFS based volume
 type Volume struct {
 
-	// This is the nfs host the the volume belongs to
+	// VS Host + Path to the actual volume
 	Host *url.URL
 
-	// Path on the Host where the volume is located
+	// Path of the volume from the volumestore target
 	Path string
 }
 
 func NewVolume(host *url.URL, NFSPath string) Volume {
+	diskLocation := &url.URL{
```

Comment:
Minor.  We're naming challenged.  However, lets not conflate disks with nfs targets.  This isn't a disk.